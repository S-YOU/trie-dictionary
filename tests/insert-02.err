nodes: 2
nodes: 3
nodes: 4
nodes: 5
nodes: 7
nodes: 9
nodes: 10
nodes: 11
nodes: 13
nodes: 15
nodes: 16
nodes: 18
nodes: 20
nodes: 21
nodes: 22
nodes: 23
nodes: 25
nodes: 26
nodes: 27
nodes: 29
nodes: 31
nodes: 32
nodes: 33
nodes: 34
nodes: 36
nodes: 38
nodes: 39
nodes: 41
nodes: 42
nodes: 44
nodes: 46
nodes: 48
nodes: 49
nodes: 50
nodes: 52
nodes: 54
nodes: 56
nodes: 57
nodes: 58
nodes: 59
nodes: 60
nodes: 61
nodes: 63
nodes: 64
nodes: 66
nodes: 67
nodes: 69
nodes: 71
nodes: 72
nodes: 74
nodes: 76
nodes: 77
nodes: 78
nodes: 80
nodes: 82
nodes: 83
nodes: 84
nodes: 86
nodes: 87
nodes: 89
nodes: 90
nodes: 92
nodes: 94
nodes: 95
nodes: 97
nodes: 99
nodes: 101
nodes: 103
nodes: 105
nodes: 107
nodes: 108
nodes: 110
nodes: 111
nodes: 113
nodes: 114
nodes: 116
nodes: 117
nodes: 118
nodes: 120
nodes: 121
nodes: 123
nodes: 125
nodes: 126
nodes: 128
nodes: 129
nodes: 130
nodes: 131
nodes: 132
nodes: 133
nodes: 135
nodes: 136
nodes: 137
nodes: 139
nodes: 141
nodes: 142
nodes: 144
nodes: 146
nodes: 147
nodes: 148
nodes: 150
nodes: 151
nodes: 153
nodes: 154
nodes: 155
nodes: 156
nodes: 158
nodes: 159
nodes: 160
nodes: 161
nodes: 163
nodes: 165
nodes: 167
nodes: 169
nodes: 170
nodes: 171
nodes: 173
nodes: 175
nodes: 177
nodes: 178
nodes: 180
nodes: 181
nodes: 182
nodes: 184
nodes: 186
nodes: 188
nodes: 189
nodes: 191
nodes: 192
nodes: 193
nodes: 195
nodes: 197
nodes: 198
nodes: 199
nodes: 200
nodes: 201
nodes: 202
nodes: 203
nodes: 205
nodes: 206
nodes: 207
nodes: 208
nodes: 209
nodes: 210
nodes: 212
nodes: 213
nodes: 215
nodes: 217
nodes: 219
nodes: 220
nodes: 221
nodes: 223
nodes: 224
nodes: 225
nodes: 226
nodes: 227
nodes: 228
nodes: 229
nodes: 230
nodes: 232
nodes: 233
nodes: 234
nodes: 236
nodes: 238
nodes: 239
nodes: 241
nodes: 243
nodes: 244
nodes: 246
nodes: 248
nodes: 250
nodes: 251
nodes: 253
nodes: 255
nodes: 257
nodes: 259
nodes: 260
nodes: 261
nodes: 263
nodes: 265
nodes: 266
nodes: 268
nodes: 269
nodes: 270
nodes: 271
nodes: 273
nodes: 274
nodes: 275
nodes: 276
nodes: 277
nodes: 279
nodes: 281
nodes: 282
nodes: 283
nodes: 285
nodes: 286
nodes: 287
nodes: 289
nodes: 290
nodes: 291
nodes: 293
nodes: 295
nodes: 296
nodes: 298
nodes: 299
nodes: 301
nodes: 303
nodes: 305
nodes: 307
nodes: 309
nodes: 311
nodes: 313
nodes: 315
nodes: 316
nodes: 317
nodes: 319
nodes: 320
nodes: 322
nodes: 323
nodes: 324
nodes: 325
nodes: 326
nodes: 327
nodes: 328
nodes: 330
nodes: 332
nodes: 334
nodes: 335
nodes: 337
nodes: 339
nodes: 340
nodes: 342
nodes: 344
nodes: 345
nodes: 347
nodes: 348
nodes: 350
nodes: 352
nodes: 353
nodes: 355
nodes: 357
nodes: 359
nodes: 360
nodes: 361
nodes: 362
nodes: 363
nodes: 365
nodes: 366
nodes: 366
nodes: 367
nodes: 368
nodes: 370
nodes: 371
nodes: 372
nodes: 374
nodes: 376
nodes: 377
nodes: 378
nodes: 380
nodes: 381
nodes: 382
nodes: 383
nodes: 385
nodes: 386
nodes: 387
nodes: 388
nodes: 390
nodes: 391
nodes: 392
nodes: 394
nodes: 396
nodes: 398
nodes: 399
nodes: 400
nodes: 401
nodes: 402
nodes: 404
nodes: 406
nodes: 407
nodes: 409
nodes: 411
nodes: 412
nodes: 413
nodes: 414
nodes: 415
nodes: 416
nodes: 417
nodes: 419
nodes: 421
nodes: 422
nodes: 424
nodes: 425
nodes: 426
nodes: 428
nodes: 429
nodes: 431
nodes: 433
nodes: 435
nodes: 437
nodes: 439
nodes: 441
nodes: 442
nodes: 443
nodes: 444
nodes: 445
nodes: 446
nodes: 448
nodes: 450
nodes: 451
nodes: 452
nodes: 453
nodes: 455
nodes: 456
nodes: 457
nodes: 458
nodes: 460
nodes: 461
nodes: 462
nodes: 464
nodes: 466
nodes: 468
nodes: 469
nodes: 471
nodes: 472
nodes: 473
nodes: 475
nodes: 477
nodes: 478
nodes: 480
nodes: 481
nodes: 483
nodes: 484
nodes: 486
nodes: 488
nodes: 489
nodes: 490
nodes: 491
nodes: 493
nodes: 495
nodes: 497
nodes: 499
nodes: 500
nodes: 501
nodes: 502
nodes: 504
nodes: 505
nodes: 506
nodes: 507
nodes: 509
nodes: 511
nodes: 512
nodes: 514
nodes: 515
nodes: 517
nodes: 518
nodes: 519
nodes: 520
nodes: 522
nodes: 523
nodes: 525
nodes: 526
nodes: 527
nodes: 528
nodes: 529
nodes: 531
nodes: 532
nodes: 534
nodes: 536
nodes: 537
nodes: 538
nodes: 540
nodes: 541
nodes: 543
nodes: 545
nodes: 546
nodes: 547
nodes: 549
nodes: 550
nodes: 551
nodes: 553
nodes: 554
nodes: 555
nodes: 556
nodes: 558
nodes: 559
nodes: 560
nodes: 561
nodes: 562
nodes: 564
nodes: 566
nodes: 567
nodes: 568
nodes: 570
nodes: 571
nodes: 572
nodes: 573
nodes: 575
nodes: 576
nodes: 578
nodes: 580
nodes: 582
nodes: 584
nodes: 585
nodes: 587
nodes: 588
nodes: 590
nodes: 591
nodes: 592
nodes: 593
nodes: 594
nodes: 595
nodes: 596
nodes: 597
nodes: 599
nodes: 601
nodes: 603
nodes: 604
nodes: 605
nodes: 606
nodes: 608
nodes: 610
nodes: 612
nodes: 614
nodes: 615
nodes: 617
nodes: 618
nodes: 619
nodes: 621
nodes: 622
nodes: 624
nodes: 626
nodes: 628
nodes: 629
nodes: 630
nodes: 632
nodes: 634
nodes: 635
nodes: 636
nodes: 637
nodes: 638
nodes: 640
nodes: 642
nodes: 643
nodes: 645
nodes: 646
nodes: 647
nodes: 648
nodes: 650
nodes: 652
nodes: 653
nodes: 654
nodes: 655
nodes: 656
nodes: 658
nodes: 659
nodes: 660
nodes: 662
nodes: 664
nodes: 665
nodes: 667
nodes: 668
nodes: 670
nodes: 671
nodes: 673
nodes: 674
nodes: 676
nodes: 677
nodes: 679
nodes: 681
nodes: 682
nodes: 684
nodes: 685
nodes: 686
nodes: 688
nodes: 690
nodes: 691
nodes: 693
nodes: 694
nodes: 695
nodes: 697
nodes: 698
nodes: 699
nodes: 701
nodes: 702
nodes: 704
nodes: 705
nodes: 707
nodes: 708
nodes: 709
nodes: 711
nodes: 713
nodes: 715
nodes: 716
nodes: 717
nodes: 719
nodes: 720
nodes: 721
nodes: 723
nodes: 724
nodes: 726
nodes: 727
nodes: 728
nodes: 729
nodes: 731
nodes: 733
nodes: 734
nodes: 736
nodes: 737
nodes: 738
nodes: 740
nodes: 742
nodes: 744
nodes: 746
nodes: 748
nodes: 750
nodes: 751
nodes: 753
nodes: 755
nodes: 757
nodes: 759
nodes: 760
nodes: 761
nodes: 763
nodes: 764
nodes: 766
nodes: 767
nodes: 769
nodes: 770
nodes: 771
nodes: 772
nodes: 774
nodes: 775
nodes: 776
nodes: 778
nodes: 779
nodes: 780
nodes: 781
nodes: 782
nodes: 783
nodes: 785
nodes: 786
nodes: 788
nodes: 789
nodes: 790
nodes: 791
nodes: 792
nodes: 794
nodes: 796
nodes: 798
nodes: 799
nodes: 801
nodes: 803
nodes: 804
nodes: 805
nodes: 806
nodes: 807
nodes: 808
nodes: 810
nodes: 811
nodes: 813
nodes: 814
nodes: 816
nodes: 817
nodes: 818
nodes: 820
nodes: 821
nodes: 822
nodes: 823
nodes: 825
nodes: 827
nodes: 828
nodes: 829
nodes: 831
nodes: 833
nodes: 835
nodes: 836
nodes: 837
nodes: 838
nodes: 839
nodes: 841
nodes: 843
nodes: 845
nodes: 846
nodes: 848
nodes: 849
nodes: 850
nodes: 852
nodes: 854
nodes: 855
nodes: 857
nodes: 858
nodes: 859
nodes: 861
nodes: 863
nodes: 865
nodes: 867
nodes: 868
nodes: 870
nodes: 871
nodes: 872
nodes: 874
nodes: 876
nodes: 878
nodes: 879
nodes: 880
nodes: 881
nodes: 883
nodes: 884
nodes: 885
nodes: 886
nodes: 888
nodes: 890
nodes: 891
nodes: 892
nodes: 893
nodes: 895
nodes: 896
nodes: 898
nodes: 899
nodes: 901
nodes: 902
nodes: 903
nodes: 904
nodes: 905
nodes: 906
nodes: 907
nodes: 909
nodes: 910
nodes: 912
nodes: 913
nodes: 914
nodes: 915
nodes: 917
nodes: 919
nodes: 920
nodes: 921
nodes: 922
nodes: 923
nodes: 924
nodes: 926
nodes: 927
nodes: 929
nodes: 931
nodes: 933
nodes: 935
nodes: 936
nodes: 938
nodes: 940
nodes: 942
nodes: 944
nodes: 945
nodes: 947
nodes: 948
nodes: 950
nodes: 951
nodes: 952
nodes: 954
nodes: 956
nodes: 957
nodes: 959
nodes: 960
nodes: 961
nodes: 963
nodes: 964
nodes: 966
nodes: 967
nodes: 968
nodes: 970
nodes: 971
nodes: 972
nodes: 973
nodes: 974
nodes: 975
nodes: 977
nodes: 979
nodes: 981
nodes: 983
nodes: 984
nodes: 986
nodes: 987
nodes: 989
nodes: 991
nodes: 993
nodes: 995
nodes: 996
nodes: 997
nodes: 999
nodes: 1000
nodes: 1002
nodes: 1003
nodes: 1004
nodes: 1005
nodes: 1007
nodes: 1008
nodes: 1010
nodes: 1011
nodes: 1013
nodes: 1014
nodes: 1015
nodes: 1016
nodes: 1017
nodes: 1018
nodes: 1019
nodes: 1021
nodes: 1022
nodes: 1024
nodes: 1026
nodes: 1027
nodes: 1028
nodes: 1029
nodes: 1030
nodes: 1031
nodes: 1033
nodes: 1035
nodes: 1037
nodes: 1038
nodes: 1040
nodes: 1042
