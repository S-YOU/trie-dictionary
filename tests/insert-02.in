insert bulbasaur
insert ivysaur
insert venusaur
insert charmander
insert charmeleon
insert charizard
insert squirtle
insert wartortle
insert blastoise
insert caterpie
insert metapod
insert butterfree
insert weedle
insert kakuna
insert beedrill
insert pidgey
insert pidgeotto
insert pidgeot
insert rattata
insert raticate
insert spearow
insert fearow
insert ekans
insert arbok
insert pikachu
insert raichu
insert sandshrew
insert sandslash
insert nidoran
insert nidorina
insert nidoqueen
insert nidoran
insert nidorino
insert nidoking
insert clefairy
insert clefable
insert vulpix
insert ninetales
insert jigglypuff
insert wigglytuff
insert zubat
insert golbat
insert oddish
insert gloom
insert vileplume
insert paras
insert parasect
insert venonat
insert venomoth
insert diglett
insert dugtrio
insert meowth
insert persian
insert psyduck
insert golduck
insert mankey
insert primeape
insert growlithe
insert arcanine
insert poliwag
insert poliwhirl
insert poliwrath
insert abra
insert kadabra
insert alakazam
insert machop
insert machoke
insert machamp
insert bellsprout
insert weepinbell
insert victreebel
insert tentacool
insert tentacruel
insert geodude
insert graveler
insert golem
insert ponyta
insert rapidash
insert slowpoke
insert slowbro
insert magnemite
insert magneton
insert farfetch
insert doduo
insert dodrio
insert seel
insert dewgong
insert grimer
insert muk
insert shellder
insert cloyster
insert gastly
insert haunter
insert gengar
insert onix
insert drowzee
insert hypno
insert krabby
insert kingler
insert voltorb
insert electrode
insert exeggcute
insert exeggutor
insert cubone
insert marowak
insert hitmonlee
insert hitmonchan
insert lickitung
insert koffing
insert weezing
insert rhyhorn
insert rhydon
insert chansey
insert tangela
insert kangaskhan
insert horsea
insert seadra
insert goldeen
insert seaking
insert staryu
insert starmie
insert mrmime
insert scyther
insert jynx
insert electabuzz
insert magmar
insert pinsir
insert tauros
insert magikarp
insert gyarados
insert lapras
insert ditto
insert eevee
insert vaporeon
insert jolteon
insert flareon
insert porygon
insert omanyte
insert omastar
insert kabuto
insert kabutops
insert aerodactyl
insert snorlax
insert articuno
insert zapdos
insert moltres
insert dratini
insert dragonair
insert dragonite
insert mewtwo
insert mew
insert chikorita
insert bayleef
insert meganium
insert cyndaquil
insert quilava
insert typhlosion
insert totodile
insert croconaw
insert feraligatr
insert sentret
insert furret
insert hoothoot
insert noctowl
insert ledyba
insert ledian
insert spinarak
insert ariados
insert crobat
insert chinchou
insert lanturn
insert pichu
insert cleffa
insert igglybuff
insert togepi
insert togetic
insert natu
insert xatu
insert mareep
insert flaaffy
insert ampharos
insert bellossom
insert marill
insert azumarill
insert sudowoodo
insert politoed
insert hoppip
insert skiploom
insert jumpluff
insert aipom
insert sunkern
insert sunflora
insert yanma
insert wooper
insert quagsire
insert espeon
insert umbreon
insert murkrow
insert slowking
insert misdreavus
insert unown
insert wobbuffet
insert girafarig
insert pineco
insert forretress
insert dunsparce
insert gligar
insert steelix
insert snubbull
insert granbull
insert qwilfish
insert scizor
insert shuckle
insert heracross
insert sneasel
insert teddiursa
insert ursaring
insert slugma
insert magcargo
insert swinub
insert piloswine
insert corsola
insert remoraid
insert octillery
insert delibird
insert mantine
insert skarmory
insert houndour
insert houndoom
insert kingdra
insert phanpy
insert donphan
insert porygon2
insert stantler
insert smeargle
insert tyrogue
insert hitmontop
insert smoochum
insert elekid
insert magby
insert miltank
insert blissey
insert raikou
insert entei
insert suicune
insert larvitar
insert pupitar
insert tyranitar
insert lugia
insert ho
insert celebi
insert treecko
insert grovyle
insert sceptile
insert torchic
insert combusken
insert blaziken
insert mudkip
insert marshtomp
insert swampert
insert poochyena
insert mightyena
insert zigzagoon
insert linoone
insert wurmple
insert silcoon
insert beautifly
insert cascoon
insert dustox
insert lotad
insert lombre
insert ludicolo
insert seedot
insert nuzleaf
insert shiftry
insert taillow
insert swellow
insert wingull
insert pelipper
insert ralts
insert kirlia
insert gardevoir
insert surskit
insert masquerain
insert shroomish
insert breloom
insert slakoth
insert vigoroth
insert slaking
insert nincada
insert ninjask
insert shedinja
insert whismur
insert loudred
insert exploud
insert makuhita
insert hariyama
insert azurill
insert nosepass
insert skitty
insert delcatty
insert sableye
insert mawile
insert aron
insert lairon
insert aggron
insert meditite
insert medicham
insert electrike
insert manectric
insert plusle
insert minun
insert volbeat
insert illumise
insert roselia
insert gulpin
insert swalot
insert carvanha
insert sharpedo
insert wailmer
insert wailord
insert numel
insert camerupt
insert torkoal
insert spoink
insert grumpig
insert spinda
insert trapinch
insert vibrava
insert flygon
insert cacnea
insert cacturne
insert swablu
insert altaria
insert zangoose
insert seviper
insert lunatone
insert solrock
insert barboach
insert whiscash
insert corphish
insert crawdaunt
insert baltoy
insert claydol
insert lileep
insert cradily
insert anorith
insert armaldo
insert feebas
insert milotic
insert castform
insert kecleon
insert shuppet
insert banette
insert duskull
insert dusclops
insert tropius
insert chimecho
insert absol
insert wynaut
insert snorunt
insert glalie
insert spheal
insert sealeo
insert walrein
insert clamperl
insert huntail
insert gorebyss
insert relicanth
insert luvdisc
insert bagon
insert shelgon
insert salamence
insert beldum
insert metang
insert metagross
insert regirock
insert regice
insert registeel
insert latias
insert latios
insert kyogre
insert groudon
insert rayquaza
insert jirachi
insert deoxys
insert turtwig
insert grotle
insert torterra
insert chimchar
insert monferno
insert infernape
insert piplup
insert prinplup
insert empoleon
insert starly
insert staravia
insert staraptor
insert bidoof
insert bibarel
insert kricketot
insert kricketune
insert shinx
insert luxio
insert luxray
insert budew
insert roserade
insert cranidos
insert rampardos
insert shieldon
insert bastiodon
insert burmy
insert wormadam
insert mothim
insert combee
insert vespiquen
insert pachirisu
insert buizel
insert floatzel
insert cherubi
insert cherrim
insert shellos
insert gastrodon
insert ambipom
insert drifloon
insert drifblim
insert buneary
insert lopunny
insert mismagius
insert honchkrow
insert glameow
insert purugly
insert chingling
insert stunky
insert skuntank
insert bronzor
insert bronzong
insert bonsly
insert mime
insert happiny
insert chatot
insert spiritomb
insert gible
insert gabite
insert garchomp
insert munchlax
insert riolu
insert lucario
insert hippopotas
insert hippowdon
insert skorupi
insert drapion
insert croagunk
insert toxicroak
insert carnivine
insert finneon
insert lumineon
insert mantyke
insert snover
insert abomasnow
insert weavile
insert magnezone
insert lickilicky
insert rhyperior
insert tangrowth
insert electivire
insert magmortar
insert togekiss
insert yanmega
insert leafeon
insert glaceon
insert gliscor
insert mamoswine
insert porygon
insert gallade
insert probopass
insert dusknoir
insert froslass
insert rotom
insert uxie
insert mesprit
insert azelf
insert dialga
insert palkia
insert heatran
insert regigigas
insert giratina
insert cresselia
insert phione
insert manaphy
insert darkrai
insert shaymin
insert arceus
insert victini
insert snivy
insert servine
insert serperior
insert tepig
insert pignite
insert emboar
insert oshawott
insert dewott
insert samurott
insert patrat
insert watchog
insert lillipup
insert herdier
insert stoutland
insert purrloin
insert liepard
insert pansage
insert simisage
insert pansear
insert simisear
insert panpour
insert simipour
insert munna
insert musharna
insert pidove
insert tranquill
insert unfezant
insert blitzle
insert zebstrika
insert roggenrola
insert boldore
insert gigalith
insert woobat
insert swoobat
insert drilbur
insert excadrill
insert audino
insert timburr
insert gurdurr
insert conkeldurr
insert tympole
insert palpitoad
insert seismitoad
insert throh
insert sawk
insert sewaddle
insert swadloon
insert leavanny
insert venipede
insert whirlipede
insert scolipede
insert cottonee
insert whimsicott
insert petilil
insert lilligant
insert basculin
insert sandile
insert krokorok
insert krookodile
insert darumaka
insert darmanitan
insert maractus
insert dwebble
insert crustle
insert scraggy
insert scrafty
insert sigilyph
insert yamask
insert cofagrigus
insert tirtouga
insert carracosta
insert archen
insert archeops
insert trubbish
insert garbodor
insert zorua
insert zoroark
insert minccino
insert cinccino
insert gothita
insert gothorita
insert gothitelle
insert solosis
insert duosion
insert reuniclus
insert ducklett
insert swanna
insert vanillite
insert vanillish
insert vanilluxe
insert deerling
insert sawsbuck
insert emolga
insert karrablast
insert escavalier
insert foongus
insert amoonguss
insert frillish
insert jellicent
insert alomomola
insert joltik
insert galvantula
insert ferroseed
insert ferrothorn
insert klink
insert klang
insert klinklang
insert tynamo
insert eelektrik
insert eelektross
insert elgyem
insert beheeyem
insert litwick
insert lampent
insert chandelure
insert axew
insert fraxure
insert haxorus
insert cubchoo
insert beartic
insert cryogonal
insert shelmet
insert accelgor
insert stunfisk
insert mienfoo
insert mienshao
insert druddigon
insert golett
insert golurk
insert pawniard
insert bisharp
insert bouffalant
insert rufflet
insert braviary
insert vullaby
insert mandibuzz
insert heatmor
insert durant
insert deino
insert zweilous
insert hydreigon
insert larvesta
insert volcarona
insert cobalion
insert terrakion
insert virizion
insert tornadus
insert thundurus
insert reshiram
insert zekrom
insert landorus
insert kyurem
insert keldeo
insert meloetta
insert genesect
insert chespin
insert quilladin
insert chesnaught
insert fennekin
insert braixen
insert delphox
insert froakie
insert frogadier
insert greninja
insert bunnelby
insert diggersby
insert fletchling
insert fletchinder
insert talonflame
insert scatterbug
insert spewpa
insert vivillon
insert litleo
insert pyroar
insert flabebe
insert floette
insert florges
insert skiddo
insert gogoat
insert pancham
insert pangoro
insert furfrou
insert espurr
insert meowstic
insert honedge
insert doublade
insert aegislash
insert spritzee
insert aromatisse
insert swirlix
insert slurpuff
insert inkay
insert malamar
insert binacle
insert barbaracle
insert skrelp
insert dragalge
insert clauncher
insert clawitzer
insert helioptile
insert heliolisk
insert tyrunt
insert tyrantrum
insert amaura
insert aurorus
insert sylveon
insert hawlucha
insert dedenne
insert carbink
insert goomy
insert sliggoo
insert goodra
insert klefki
insert phantump
insert trevenant
insert pumpkaboo
insert gourgeist
insert bergmite
insert avalugg
insert noibat
insert noivern
insert xerneas
insert yveltal
insert zygarde
insert diancie
insert hoopa
