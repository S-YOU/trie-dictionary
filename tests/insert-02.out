word number: 0
word number: 1
word number: 2
word number: 3
word number: 4
word number: 5
word number: 6
word number: 7
word number: 8
word number: 9
word number: 10
word number: 11
word number: 12
word number: 13
word number: 14
word number: 15
word number: 16
word number: 17
word number: 18
word number: 19
word number: 20
word number: 21
word number: 22
word number: 23
word number: 24
word number: 25
word number: 26
word number: 27
word number: 28
word number: 29
word number: 30
ignored
word number: 31
word number: 32
word number: 33
word number: 34
word number: 35
word number: 36
word number: 37
word number: 38
word number: 39
word number: 40
word number: 41
word number: 42
word number: 43
word number: 44
word number: 45
word number: 46
word number: 47
word number: 48
word number: 49
word number: 50
word number: 51
word number: 52
word number: 53
word number: 54
word number: 55
word number: 56
word number: 57
word number: 58
word number: 59
word number: 60
word number: 61
word number: 62
word number: 63
word number: 64
word number: 65
word number: 66
word number: 67
word number: 68
word number: 69
word number: 70
word number: 71
word number: 72
word number: 73
word number: 74
word number: 75
word number: 76
word number: 77
word number: 78
word number: 79
word number: 80
word number: 81
word number: 82
word number: 83
word number: 84
word number: 85
word number: 86
word number: 87
word number: 88
word number: 89
word number: 90
word number: 91
word number: 92
word number: 93
word number: 94
word number: 95
word number: 96
word number: 97
word number: 98
word number: 99
word number: 100
word number: 101
word number: 102
word number: 103
word number: 104
word number: 105
word number: 106
word number: 107
word number: 108
word number: 109
word number: 110
word number: 111
word number: 112
word number: 113
word number: 114
word number: 115
word number: 116
word number: 117
word number: 118
word number: 119
word number: 120
word number: 121
word number: 122
word number: 123
word number: 124
word number: 125
word number: 126
word number: 127
word number: 128
word number: 129
word number: 130
word number: 131
word number: 132
word number: 133
word number: 134
word number: 135
word number: 136
word number: 137
word number: 138
word number: 139
word number: 140
word number: 141
word number: 142
word number: 143
word number: 144
word number: 145
word number: 146
word number: 147
word number: 148
word number: 149
word number: 150
word number: 151
word number: 152
word number: 153
word number: 154
word number: 155
word number: 156
word number: 157
word number: 158
word number: 159
word number: 160
word number: 161
word number: 162
word number: 163
word number: 164
word number: 165
word number: 166
word number: 167
word number: 168
word number: 169
word number: 170
word number: 171
word number: 172
word number: 173
word number: 174
word number: 175
word number: 176
word number: 177
word number: 178
word number: 179
word number: 180
word number: 181
word number: 182
word number: 183
word number: 184
word number: 185
word number: 186
word number: 187
word number: 188
word number: 189
word number: 190
word number: 191
word number: 192
word number: 193
word number: 194
word number: 195
word number: 196
word number: 197
word number: 198
word number: 199
word number: 200
word number: 201
word number: 202
word number: 203
word number: 204
word number: 205
word number: 206
word number: 207
word number: 208
word number: 209
word number: 210
word number: 211
word number: 212
word number: 213
word number: 214
word number: 215
word number: 216
word number: 217
word number: 218
word number: 219
word number: 220
word number: 221
word number: 222
word number: 223
word number: 224
word number: 225
word number: 226
word number: 227
word number: 228
word number: 229
word number: 230
ignored
word number: 231
word number: 232
word number: 233
word number: 234
word number: 235
word number: 236
word number: 237
word number: 238
word number: 239
word number: 240
word number: 241
word number: 242
word number: 243
word number: 244
word number: 245
word number: 246
word number: 247
word number: 248
word number: 249
word number: 250
word number: 251
word number: 252
word number: 253
word number: 254
word number: 255
word number: 256
word number: 257
word number: 258
word number: 259
word number: 260
word number: 261
word number: 262
word number: 263
word number: 264
word number: 265
word number: 266
word number: 267
word number: 268
word number: 269
word number: 270
word number: 271
word number: 272
word number: 273
word number: 274
word number: 275
word number: 276
word number: 277
word number: 278
word number: 279
word number: 280
word number: 281
word number: 282
word number: 283
word number: 284
word number: 285
word number: 286
word number: 287
word number: 288
word number: 289
word number: 290
word number: 291
word number: 292
word number: 293
word number: 294
word number: 295
word number: 296
word number: 297
word number: 298
word number: 299
word number: 300
word number: 301
word number: 302
word number: 303
word number: 304
word number: 305
word number: 306
word number: 307
word number: 308
word number: 309
word number: 310
word number: 311
word number: 312
word number: 313
word number: 314
word number: 315
word number: 316
word number: 317
word number: 318
word number: 319
word number: 320
word number: 321
word number: 322
word number: 323
word number: 324
word number: 325
word number: 326
word number: 327
word number: 328
word number: 329
word number: 330
word number: 331
word number: 332
word number: 333
word number: 334
word number: 335
word number: 336
word number: 337
word number: 338
word number: 339
word number: 340
word number: 341
word number: 342
word number: 343
word number: 344
word number: 345
word number: 346
word number: 347
word number: 348
word number: 349
word number: 350
word number: 351
word number: 352
word number: 353
word number: 354
word number: 355
word number: 356
word number: 357
word number: 358
word number: 359
word number: 360
word number: 361
word number: 362
word number: 363
word number: 364
word number: 365
word number: 366
word number: 367
word number: 368
word number: 369
word number: 370
word number: 371
word number: 372
word number: 373
word number: 374
word number: 375
word number: 376
word number: 377
word number: 378
word number: 379
word number: 380
word number: 381
word number: 382
word number: 383
word number: 384
word number: 385
word number: 386
word number: 387
word number: 388
word number: 389
word number: 390
word number: 391
word number: 392
word number: 393
word number: 394
word number: 395
word number: 396
word number: 397
word number: 398
word number: 399
word number: 400
word number: 401
word number: 402
word number: 403
word number: 404
word number: 405
word number: 406
word number: 407
word number: 408
word number: 409
word number: 410
word number: 411
word number: 412
word number: 413
word number: 414
word number: 415
word number: 416
word number: 417
word number: 418
word number: 419
word number: 420
word number: 421
word number: 422
word number: 423
word number: 424
word number: 425
word number: 426
word number: 427
word number: 428
word number: 429
word number: 430
word number: 431
word number: 432
word number: 433
word number: 434
word number: 435
word number: 436
word number: 437
word number: 438
word number: 439
word number: 440
word number: 441
word number: 442
word number: 443
word number: 444
word number: 445
word number: 446
word number: 447
word number: 448
word number: 449
word number: 450
word number: 451
word number: 452
word number: 453
word number: 454
word number: 455
word number: 456
word number: 457
word number: 458
word number: 459
word number: 460
word number: 461
word number: 462
word number: 463
word number: 464
word number: 465
word number: 466
word number: 467
word number: 468
word number: 469
word number: 470
ignored
word number: 471
word number: 472
word number: 473
word number: 474
word number: 475
word number: 476
word number: 477
word number: 478
word number: 479
word number: 480
word number: 481
word number: 482
word number: 483
word number: 484
word number: 485
word number: 486
word number: 487
word number: 488
word number: 489
word number: 490
word number: 491
word number: 492
word number: 493
word number: 494
word number: 495
word number: 496
word number: 497
word number: 498
word number: 499
word number: 500
word number: 501
word number: 502
word number: 503
word number: 504
word number: 505
word number: 506
word number: 507
word number: 508
word number: 509
word number: 510
word number: 511
word number: 512
word number: 513
word number: 514
word number: 515
word number: 516
word number: 517
word number: 518
word number: 519
word number: 520
word number: 521
word number: 522
word number: 523
word number: 524
word number: 525
word number: 526
word number: 527
word number: 528
word number: 529
word number: 530
word number: 531
word number: 532
word number: 533
word number: 534
word number: 535
word number: 536
word number: 537
word number: 538
word number: 539
word number: 540
word number: 541
word number: 542
word number: 543
word number: 544
word number: 545
word number: 546
word number: 547
word number: 548
word number: 549
word number: 550
word number: 551
word number: 552
word number: 553
word number: 554
word number: 555
word number: 556
word number: 557
word number: 558
word number: 559
word number: 560
word number: 561
word number: 562
word number: 563
word number: 564
word number: 565
word number: 566
word number: 567
word number: 568
word number: 569
word number: 570
word number: 571
word number: 572
word number: 573
word number: 574
word number: 575
word number: 576
word number: 577
word number: 578
word number: 579
word number: 580
word number: 581
word number: 582
word number: 583
word number: 584
word number: 585
word number: 586
word number: 587
word number: 588
word number: 589
word number: 590
word number: 591
word number: 592
word number: 593
word number: 594
word number: 595
word number: 596
word number: 597
word number: 598
word number: 599
word number: 600
word number: 601
word number: 602
word number: 603
word number: 604
word number: 605
word number: 606
word number: 607
word number: 608
word number: 609
word number: 610
word number: 611
word number: 612
word number: 613
word number: 614
word number: 615
word number: 616
word number: 617
word number: 618
word number: 619
word number: 620
word number: 621
word number: 622
word number: 623
word number: 624
word number: 625
word number: 626
word number: 627
word number: 628
word number: 629
word number: 630
word number: 631
word number: 632
word number: 633
word number: 634
word number: 635
word number: 636
word number: 637
word number: 638
word number: 639
word number: 640
word number: 641
word number: 642
word number: 643
word number: 644
word number: 645
word number: 646
word number: 647
word number: 648
word number: 649
word number: 650
word number: 651
word number: 652
word number: 653
word number: 654
word number: 655
word number: 656
word number: 657
word number: 658
word number: 659
word number: 660
word number: 661
word number: 662
word number: 663
word number: 664
word number: 665
word number: 666
word number: 667
word number: 668
word number: 669
word number: 670
word number: 671
word number: 672
word number: 673
word number: 674
word number: 675
word number: 676
word number: 677
word number: 678
word number: 679
word number: 680
word number: 681
word number: 682
word number: 683
word number: 684
word number: 685
word number: 686
word number: 687
word number: 688
word number: 689
word number: 690
word number: 691
word number: 692
word number: 693
word number: 694
word number: 695
word number: 696
word number: 697
word number: 698
word number: 699
word number: 700
word number: 701
word number: 702
word number: 703
word number: 704
word number: 705
word number: 706
word number: 707
word number: 708
word number: 709
word number: 710
word number: 711
word number: 712
word number: 713
word number: 714
word number: 715
word number: 716
